.github/workflows/build_packages.yaml

```diff
@@ -351,15 +351,18 @@ jobs:
       fail-fast: false
       matrix:
         profile: ${{fromJSON(needs.prepare.outputs.profiles)}}
-        registry:
-          - 'docker.io'
         otp:
           - 24.1.5-3
-        include:
-          - profile: emqx
+        registry:
```

Comment:
There are a few typos below 
if: matrix.repository == 'public.ecr.aws'
should be registry instead of repository

Reply:
Thanks, it's fixed